### PR TITLE
Pbdev

### PR DESCRIPTION
**Pull request by Pavan-Bellam**
Supporting extraction of only specific attributes

### CODE REVIEW

File: modules/png-extraction/config.json [hunk 1]
```diff
@@ -6,9 +6,11 @@
 	"PrintImages": true,
 	"CommonHeadersOnly": false,
 	"PublicHeadersOnly": true,
+	"SpecificHeadersOnly": false
```
Comment (pradeeban) [inline modules/png-extraction/config.json:9]: It should be,
"SpecificHeadersOnly": false,

The missing of "," throws,

Traceback (most recent call last):
File "ImageExtractor.py", line 525, in
niffler = json.load(f)
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/init.py", line 296, in load
parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/init.py", line 348, in loads
return _default_decoder.decode(s)
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
obj, end = self.raw_decode(s, idx=_w(s, 0).end())
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 353, in raw_decode
obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 10 column 2 (char 248)

Please fix.

File: modules/png-extraction/config.json [hunk 2]
```diff
@@ -6,9 +6,11 @@
...
 	"UseProcesses": 0,
 	"FlattenedToLevel": "patient",
 	"is16Bit":true,
 	"SendEmail": true,
-	"YourEmail": "test@test.test"
+	"YourEmail": "test@test.test",
```
Comment (pradeeban) [inline modules/png-extraction/config.json:14]: and there should be no "," here.

It, as of now, also would throw:

Traceback (most recent call last):
File "ImageExtractor.py", line 525, in
niffler = json.load(f)
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/init.py", line 296, in load
parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/init.py", line 348, in loads
return _default_decoder.decode(s)
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
obj, end = self.raw_decode(s, idx=_w(s, 0).end())
File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 353, in raw_decode
obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 16 column 1 (char 370)

**Comment by pradeeban** [comment]
Made 2 fixes to config.json. It was just a misplaced comma. With that fix, it works.

**Comment by pradeeban** [comment]
Fixes #282

**Comment by Pavan-Bellam** [comment]
@pradeeban Thank you! looking forward to contribute more.

**Comment by pradeeban** [comment]
Thanks a lot for your valuable contributions, @Pavan-Bellam. 👍

**Pull request merged**